azuread/table_azuread_group.go

```diff
@@ -112,24 +112,16 @@ func listAdGroups(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateDat
 		input.Filter = strings.Join(filter, " and ")
 	}
 
-	// if input.Filter != "" {
-	// 	plugin.Logger(ctx).Debug("Filter", "input.Filter", input.Filter)
-	// }
-
-	pagesLeft := true
```

Comment:
We removed this paging code since it didn't work, but do we have any support for paging afterward?